src/engine/ruleset/decoders/windows/windows-security.yml

```diff
@@ -0,0 +1,326 @@
+name: decoder/windows-security/0
+
+sources: 
+  - decoder/windows-event-decoder/0
+
+metadata:
+  description: Decoder for Windows Security events
+  module: Windows Security
+  title: Decoder for Windows Security and Windows Eventlog auditing
+  compatibility: >
+    This decoder has been tested on Wazuh version 4.3
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/23
+  references:
+    - https://learn.microsoft.com/en-us/windows/security/threat-protection/auditing/security-auditing-overview
+
+check: ~windows.Event.System.Provider.@Name==Microsoft-Windows-Security-Auditing OR ~windows.Event.System.Provider.@Name==Microsoft-Windows-Eventlog
+
+
+normalize:
+ - map:
+    - event.module: security
+      
+    # Set categorization fields
+    - event: +kvdb_get_merge/win_security_categories/$~windows.Event.System.EventID.#text
+    
+    # Set Logon Type
+    - winlog.logon.type: +kvdb_get/win_logon_types/$~windows.Event.EventData.LogonType
+    
+    # Set User Account Control
+    # NewUacValue 
+    # Correct implementation needs bitwise operations - Opting to use raw value
+    #- event.UserAccountControl: $~windows.Event.EventData.NewUacValue
+    - winlog.event_data.UserAccountControl: +s_to_array/$~windows.Event.EventData.UserAccountControl/%
+    
+    # Set Kerberos Ticket Options
+    # TicketOptionsDescription - Needs bitwise operations
+
+    # Set Kerberos Encryption Types
+    - event: +kvdb_get_merge/win_security_ticket_encryption_types/$~windows.Event.EventData.TicketEncryptionType
+    
+    # Set Kerberos Ticket Status Codes
+    - winlog.event_data.StatusDescription: +kvdb_get_merge/win_security_kerberos_status_types/$~windows.Event.EventData.Status
+    
+    # Set Service Type and Name
+    # Service related fields
+    - service.name: $~windows.Event.EventData.ServiceName
+    - service.type: +kvdb_get_merge/win_security_service_types/$~windows.Event.EventData.ServiceType
```

Comment:
being so many few cases, does it make sense to create a KVDB to check this?

Reply:
what would you use instead of a kvdb?

Reply:
maybe a simple check map block like:
```
- check: 
 - $~windows.Event.EventData.ServiceType: key
 map :
 - service.type: value
```

Reply:
Done